plugins/pagerduty/plugin.spec.yaml

```diff
@@ -17,48 +19,63 @@ tags:
 - incident management
 hub_tags:
   use_cases: [alerting_and_notifications]
-  keywords: [incident response, incident]
+  keywords: [incident_response, incidents]
   features: []
+sdk: 
+  type: full 
+  version: 5 
+  user: nobody
 connection:
   api_key:
     title: API Key
     type: credential_secret_key
     description: API Key
     required: true
+    example: stRbCzL92kpAfwCkSiA9
 types:
   user:
     id:
       description: ID
       type: string
+      example: PYGDZB8
     self:
       description: URL to view object
       type: string
+      example: "https://api.pagerduty.com/users/PYGDZB8"
     name:
       description: Name
       type: string
       required: true
+      example: test user 
     email:
       description: Email
       type: string
       required: true
+      example: user1@example.com
     summary:
       description: Summary
       type: string
+      example: test summary
     color:
       description: Color
       type: string
+      example: purple
     description:
       description: Description
       type: string
+      example: test description
     job_title:
       description: Job Title
       type: string
+      example: engineer
     avatar_url:
       description: Avatar URL
       type: string
+      example: "https://secure.gravatar.com/avatar/c67b6c8ed606dec999bb064ada88cf5c.png?d=mm&r=PG"
```

Comment:
Same here is it a genuine URL?

Reply:
All of these examples were fake data or else taken from the pager duty api docs, but i'll update them to make them easier to look fake